jvm/src/main/groovy/org/asciidoctor/gradle/jvm/AbstractAsciidoctorTask.groovy

```diff
@@ -93,17 +99,53 @@ class AbstractAsciidoctorTask extends AbstractJvmModelExecTask<AsciidoctorJvmExe
     public final static Severity WARN = Severity.WARN
     public final static Severity INFO = Severity.INFO
 
-    protected final AsciidoctorJExtension asciidoctorj
     private ExecutionMode inProcess
     private Severity failureLevel = Severity.FATAL
     private final List<Object> asciidocConfigurations = []
     private final File rootDir
     private final File projectDir
     private final File execConfigurationDataFile
-    private final Function<List<Dependency>, Configuration> detachedConfigurationCreator
     private final Property<FileCollection> jvmClasspath
     private final List<Provider<File>> gemJarProviders = []
 
+    protected final MapProperty<String, Object> optionsProperty = project.objects.mapProperty(String, Object)
```

Comment:
why are these `protected` instead of `private`?

Reply:
I had run into some unknown property exception issues that I thought were resolved by this, but after testing now, it works just fine with them `private`, so I've made that change.